stake-pool/program/src/lib.rs

```diff
@@ -14,4 +14,4 @@ pub mod entrypoint;
 // Export current sdk types for downstream users building with a different sdk version
 pub use solana_program;
 
-solana_program::declare_id!("STAKEPQQL1111111111111111111111111111111111");
+solana_program::declare_id!("AYyuDZeEKcDDyqRHgM6iba7ui2Dkc9ENUpUhfMd58N8e");
```

Comment:
I've got such account name generated, but since it is one-time use only (and it took 4 hours to find one) I decided to use random program id until we finish fixing bugs in the contract.

Reply:
oh ok, great! i had mine going for about 2 hours and didn't get anything unfortunately